templates/common/infra/bicep/core/host/container-app.bicep

```diff
@@ -5,13 +5,33 @@ param tags object = {}
 param containerAppsEnvironmentName string
 param containerName string = 'main'
 param containerRegistryName string
+
+@description('Minimum number of replicas to run')
```

Comment:
Should we use the `@minValue` decorator here to enforce that the value is >= 0?

Reply:
done